<?xml version="1.0"?>

<launch>
  <!--PARAMETERS-->

  <param name="odom_frame_id" value="/odom"/>
  <param name="global_frame_id" value="/map"/>
  <param name="base_frame_id" value="desistek_saga/base_link"/>
  <!--for multiple sensors, place comma eg "r1/front_laser/scan,/beacon" -->
  <param name="sensor_source" value="/sonar"/>



</launch>
